egs/wsj/s5/steps/nnet3/get_degs.sh

```diff
@@ -13,7 +13,7 @@ cmd=run.pl
 max_copy_jobs=5  # Limit disk I/O
 
 # feature options
-feat_type=raw     # set it to 'lda' to use LDA features.
+feat_type=raw     # set it to 'delta' to use delta features.
```

Comment:
disable delta features here; this only runs on top of already-trained chain system so if we remove delta in the basic chain training scripts (which you should), it has no reason to exist here.